subhub/hub/stripe/customer.py

```diff
@@ -198,29 +203,56 @@ def run(self):
                     "cancel at period end",
                     end=self.payload.data.object.cancel_at_period_end,
                 )
-                data = self.create_data(
+                data = dict(
+                    event_id=self.payload.id,
+                    event_type="customer.subscription_cancelled",
                     uid=user_id,
                     customer_id=self.payload.data.object.customer,
-                    subscriptionId=self.payload.data.object.id,  # required by FxA
                     subscription_id=self.payload.data.object.id,
                     plan_amount=self.payload.data.object.plan.amount,
                     canceled_at=self.payload.data.object.canceled_at,
                     cancel_at=self.payload.data.object.cancel_at,
                     cancel_at_period_end=self.payload.data.object.cancel_at_period_end,
                     nickname=self.payload.data.object.plan.nickname,
-                    messageCreatedAt=int(time.time()),  # required by FxA
                     invoice_id=self.payload.data.object.latest_invoice,
-                    eventId=self.payload.id,  # required by FxA
+                    current_period_start=self.payload.data.object.current_period_start,
+                    current_period_end=self.payload.data.object.current_period_end,
                 )
                 logger.info("customer subscription cancel at period end", data=data)
-                routes = [StaticRoutes.FIREFOX_ROUTE, StaticRoutes.SALESFORCE_ROUTE]
+                routes = [StaticRoutes.SALESFORCE_ROUTE]
                 self.send_to_routes(routes, json.dumps(data))
             elif (
                 not self.payload.data.object.cancel_at_period_end
                 and self.payload.data.object.status == "active"
-                and not previous_attributes.get("cancel_at_period_end")
+                # and not previous_attributes.get("cancel_at_period_end")
```

Comment:
Comment still needed?

Reply:
yeah; let's delete

Reply:
Actually I want it there as a referece, will most likely need to add a third scenario next week pending discussions between Stephen and Desiree.

Reply:
Understood.